README.undock.md

```diff
@@ -4,110 +4,38 @@
+The `user/LoadLibrary/Arm64` branch must be used. It contains the necessary patches to build WebRTC on UWP ARM64.
 
-For example, run `gn args out\debug_x64` to build for Debug UWP X64. This creates the `out\debug_x64\args.gn` file (GN configuration for the build) and opens a text editor to edit it:
+#### Build the WebRTC library
 
-```
-target_os="winuwp"
-is_clang=true
-target_cpu="x64"
-is_debug=true
-rtc_include_tests=false
-rtc_build_examples=false
-rtc_win_video_capture_winrt=true
-```
+Follow the instructions at [winrtc/patches_for_WebRTC_org/m80/readme.md](https://github.com/microsoft/winrtc/blob/user/LoadLibrary/Arm64/patches_for_WebRTC_org/m80/readme.md) to checkout and patch the WebRTC repository. When setting up a build through `gn gen`:
+- set `target_os="winuwp"` to build WebRTC for UWP apps, `="win"` to build for Windows Desktop;
+- set `target_cpu` to the target platform (`x64`, `x86`, `arm64`, `arm`);
+- set `is_debug` to `true` for Debug or `false` for Release;
+- set up the build in this path: `out\$(Configuration)_$(Platform)[_win32]\`, for example `out\Debug_x64` (this particular shape for the folder name is not required for building WebRTC, but is used by the `Microsoft.MixedReality.Native` projects to find `webrtc.lib` when linking - see below).
 
-Close the editor. The `gn` process generates build files for `ninja` in `out\debug_x64`.
+Setup and build WebRTC for the configurations you need as indicated in the WinRTC documentation.
 
-```cmd
-ninja -C out\debug_x64
-```
-
-The library is in `out\debug_x64\obj\webrtc.lib`.
-
-In order to build for a different target CPU, e.g. ARM64, repeat the process replacing the target name in the `target_cpu` line in args.gn:
-```
-target_cpu="arm64"
-```
-
-In order to build the Win32 version, replace the `target_os="winuwp"` line with:
-```
-target_os="win"
-use_static_crt="false"
-```
-
-The release version can be built by setting `is_debug` to `false`.
-
-
-### MixedReality-WebRTC Native/Core DLLs
-
-Terminology fails us here because going forward we will split the C++ library ("native") from the low-level interop wrapper DLL, and link the latter statically with Google's libwebrtc ("core"). So there are discrepancies in naming. In particular the DLLs and VS projects are still named `Microsoft.MixedReality.WebRTC.Native.dll` even though they are unrelated with the C++ library (which doesn't exist in this branch). But "core" historically refers to the Google WebRTC + WebRTC UWP packaging, though we can probably reuse that term here.
+### MixedReality-WebRTC components
 
 #### Clone the MixedReality-WebRTC repository
 
+The MixedReality-WebRTC repository must be cloned beside the `src` folder of the WebRTC clone. E.g. if in the previous steps `gclient` was run in `C:\webrtc-m80`:

```

Comment:
That's bad. Beside `src` is already inside the `libwebrtc` repo, so we're cloning an unrelated repo inside an existing repo. We should change and clone `libwebrtc` inside `<mr-webrtc>/external/` instead.

Reply:
Logged as #272.